shaka/src/public/ShakaPlayerView.mm

```diff
@@ -300,4 +308,28 @@ - (BOOL)remakeTextCues:(BOOL)sizeChanged {
   return YES;
 }
 
+- (void) orientationChanged:(NSNotification *)note {
+  if (_imageLayer.contents == nil) {
+    return;
+  }
+  // Fit image in frame.
```

Comment:
You should move the bulk of this to a separate method (e.g. `layoutImage`) and call it from both the event listener and in `renderLoop`.

Reply:
Ok, I'll do it.

Reply:
Done